worlds.md

```diff
@@ -8,4 +8,4 @@
  2018-11-16 |    5    |    32      | [beta-version-profile](https://github.com/beta-version-profile/a-tiny-JS-world)
  2018-11-17 |    5    |    70      | [linkqwd](https://github.com/linkqwd/a-tiny-JS-world/tree/populate-world)
  2018-11-20 |    5    |     27     | [IrynaY](https://github.com/IrynaY/a-tiny-JS-world/tree/populate-world)
-
+ 2018-11-23 |    4    |     46     | [kalash14](https://github.com/kalash14/a-tiny-JS-world/tree/populate-world)
```

Comment:
Please, add a newline so additions from other contributors won't affect your addition.

It makes sense to set "Ensure empty line at the end of file" or similar option in your IDE/editor settings.

If this doesn't help and editor in use eliminates the latest newline, just add two newlines :)